image.go

```diff
@@ -70,6 +70,10 @@ func Resize(buf []byte, o ImageOptions) (Image, error) {
 	if o.NoCrop == false {
 		opts.Crop = true
 	}
+	meta, err := bimg.Metadata(buf)
```

Comment:
That makes sense.

> On Apr 4, 2016, at 1:15 PM, Tomás Aparicio <notifications@github.com> wrote:
> 
> In image.go <https://github.com/h2non/imaginary/pull/69#discussion_r58440418>:
> 
> > @@ -70,6 +70,10 @@ func Resize(buf []byte, o ImageOptions) (Image, error) {
> >  	if o.NoCrop == false {
> >  		opts.Crop = true
> >  	}
> > +	meta, err := bimg.Metadata(buf)
> Consider doing this operation only if params.Background is present.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/h2non/imaginary/pull/69/files/5ad0f96f4447e20fbdf7f55827a563e098c24942#r58440418>